src/RBACHandler.test.ts

```diff
@@ -175,16 +177,127 @@ describe('isAuthorized', () => {
     test('ERROR: Attempt to create a handler to support a new config version', async () => {
         expect(() => {
             // eslint-disable-next-line no-new
-            new RBACHandler({
-                version: 2.0,
-                groupRules: {},
-            });
+            new RBACHandler(
+                {
+                    version: 2.0,
+                    groupRules: {},
+                },
+                '4.0.1',
+            );
         }).toThrow(new Error('Configuration version does not match handler version'));
     });
 });
 
+describe('isAuthorized:Export', () => {
+    const customRBACRules: RBACConfig = {
+        version: 1.0,
+        groupRules: {
+            practitioner: {
+                operations: ['read'],
+                resources: [],
+            },
+        },
+    };
+
+    afterEach(() => {
+        customRBACRules.groupRules.practitioner = {
+            operations: ['read'],
+            resources: [],
+        };
+    });
+
+    test('TRUE; GET system Export with permission to all resources', async () => {
+        customRBACRules.groupRules.practitioner.resources = SUPPORTED_R4_RESOURCES;
+        const authZHandler: RBACHandler = new RBACHandler(customRBACRules, '4.0.1');
+        const results: boolean = authZHandler.isAuthorized({
+            accessToken: practitionerAccessToken,
+            operation: 'read',
+            exportType: 'system',
+        });
+        expect(results).toEqual(true);
+    });
+
+    test('TRUE; GET system Export with permission to all resources, in mixed order', async () => {
+        customRBACRules.groupRules.practitioner.resources = shuffle(SUPPORTED_R4_RESOURCES);

```

Comment:
`shuffle` modifies the array in place. This means that all tests after this one are actually using the shuffled array. We should create a copy of the array instead(that shuffle package has an option for that)

This doesn't cause any trouble right now, but these little things can lead to weird test bugs as more tests are added.

Reply:
good point. fixed